recorder.go

```diff
@@ -244,6 +244,23 @@ func GetLightStepAccessToken(lsTracer ot.Tracer) (string, error) {
 	}
 }
 
+type Closer interface {
+	Close() error
+}
+
+func CloseTracer(tracer ot.Tracer) error {
```

Comment:
What is being passed in is an `ot.Tracer` because that is what the user will almost always have on hand. They could typecast to a lightstep tracer (or Closer, etc), but we do that internally in `CloseTracer` as a convenience.